MobileCenterUpdates/MobileCenterUpdates/MSUpdates.m

```diff
@@ -1,18 +1,29 @@
-#import <Foundation/Foundation.h>
 #import "MSDistributionSender.h"
 #import "MSLogger.h"
 #import "MSMobileCenterInternal.h"
 #import "MSReleaseDetails.h"
 #import "MSServiceAbstractProtected.h"
 #import "MSUpdates.h"
+#import "MSUpdatesPrivate.h"
 #import "MSUpdatesInternal.h"
+#import "MSUtil.h"
+
+#import <Foundation/Foundation.h>
+#import <SafariServices/SafariServices.h>
 
 /**
  * Service storage key name.
  */
 static NSString *const kMSServiceName = @"Updates";
 
 /**
+ * The header name for update token
+ */
+static NSString *const kMSUpdtsHeaderApiToken = @"x-api-token";
```

Comment:
I think you can just remove `Update` and simply say `kMSHeaderApiToken`. Others, too.